core/src/test/java/io/github/tcdl/adapters/mock/MockAdapterTest.java

```diff
@@ -0,0 +1,67 @@
+package io.github.tcdl.adapters.mock;
+
+import static org.junit.Assert.assertNotNull;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.timeout;
+import static org.mockito.Mockito.verify;
+
+import java.util.Queue;
+import java.util.concurrent.ConcurrentLinkedQueue;
+
+import io.github.tcdl.adapters.ConsumerAdapter;
+import io.github.tcdl.adapters.ProducerAdapter;
+import io.github.tcdl.exception.ChannelException;
+import io.github.tcdl.exception.JsonConversionException;
+import io.github.tcdl.messages.Message;
+import io.github.tcdl.support.TestUtils;
+import io.github.tcdl.support.Utils;
+import org.junit.Test;
+
+/**
+ * MockAdapter class represents implementation of {@link ProducerAdapter} and {@link ConsumerAdapter}
+ * for test purposes.
+ * 
+ */
+public class MockAdapterTest {
```

Comment:
if i'm correct first case is implemented , but second case u expect NEGATIVE test for same functionality and last verify should sounds like "Verify that the message has NOT been received